content/en/profiler/profiler_troubleshooting/dotnet.md

```diff
@@ -138,17 +138,20 @@ Otherwise, turn on [debug mode][1] and [open a support ticket][2] with the debug
 
 ## Reduce overhead when using the profiler
 
-### Enabling the profiler machine-wide
+The different profilers have a fixed cpu and memory overhead. Because such a fixed overhead per profiled application exists, *the more profiled applications, the higher the overhead!*
```

Comment:
`The different profilers` may be confusing to customers, they could understand `the different runtime profilers`

We could reword into `The different profile types` with a link to .NET profile types list